blurb/blurb.py

```diff
@@ -838,24 +838,14 @@ def add():
     os.close(handle)
     atexit.register(lambda : os.unlink(tmp_path))
 
-    def init_tmp_with_template():
-        with open(tmp_path, "wt", encoding="utf-8") as f:
-            # hack:
-            # my editor likes to strip trailing whitespace from lines.
-            # normally this is a good idea.  but in the case of the template
-            # it's unhelpful.
-            # so, manually ensure there's a space at the end of the bpo line.
-            text = template
-
-            bpo_line = ".. bpo:"
-            without_space = "\n" + bpo_line + "\n"
-            with_space = "\n" + bpo_line + " \n"
-            if without_space not in text:
-                sys.exit("Can't find BPO line to ensure there's a space on the end!")
-            text = text.replace(without_space, with_space)
-            f.write(text)
+    # see of the branch name indicated BPO number
+    # makes the 'editor strips trailing whitespace from template lines'
```

Comment:
Given that your patch obviates (and subsequently removes) the hack in question, this two-line comment is meaningless.